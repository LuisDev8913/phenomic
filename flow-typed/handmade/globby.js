declare module "globby" {
  declare type pattern = string | Array<string>;
  declare type async = (
    pattern: pattern,
    options: Object
  ) => Promise<Array<string>>;
  declare module.exports: {
    (): async,
    sync: (pattern: pattern, options: Object) => Array<string>
  };
}
