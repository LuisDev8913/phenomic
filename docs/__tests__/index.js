import globby from "globby";

it("should build docs correctly", () => {
  const testFolder = __dirname + "/../dist";
  const files = globby.sync("**/*", {
    cwd: testFolder,
    nodir: true
  });

  // should have html files
  expect(files.filter(file => file.endsWith(".html"))).toMatchSnapshot();
});
