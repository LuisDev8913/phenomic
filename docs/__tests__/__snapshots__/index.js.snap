// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should build docs correctly 1`] = `
Array [
  "404.html",
  "docs/getting-started/index.html",
  "index.html",
  "news/2017/06/introducing-1.0.0-alpha/index.html",
  "news/after/MjAxNy8wNi9pbnRyb2R1Y2luZy0xLjAuMC1hbHBoYQ==/index.html",
  "news/index.html",
  "showcase/after/a2hvYW5ndXllbi5tZQ==/index.html",
  "showcase/after/a2V2aW45NDA3MjYuZ2l0aHViLmlv/index.html",
  "showcase/after/aGF5d2lyZXouY29t/index.html",
  "showcase/after/aXN0YW5idWwuanMub3Jn/index.html",
  "showcase/after/bm9yZHNvZnR3YXJlLmdpdGh1Yi5pbw==/index.html",
  "showcase/after/bmhha2hvYXBodW9jYmluaC5jb20=/index.html",
  "showcase/after/bW9vbm1haWwuaW9fX2FtYXpvbl9fc2VzX19lbWFpbF9fbWFya2V0aW5n/index.html",
  "showcase/after/c2NhbGVjb25mY28uY29t/index.html",
  "showcase/after/c2VhYWRlMjAxNi52bg==/index.html",
  "showcase/after/c2VydmVybGVzcy5jb20=/index.html",
  "showcase/after/c3R5bGVsaW50Lmlv/index.html",
  "showcase/after/cG9zdGNzcy5vcmc=/index.html",
  "showcase/after/cGhlbm9taWMuaW8=/index.html",
  "showcase/after/cHV0YWluZGVjb2RlLmlv/index.html",
  "showcase/after/cmVhY3RjbGFzcy5jb20=/index.html",
  "showcase/after/d3d3LmFncmlzdGEuY29t/index.html",
  "showcase/after/d3d3LmJsdWVqYXl3aXJlbGVzcy5jb20=/index.html",
  "showcase/after/d3d3LmJvc2NuZWdyZS12YWNhbmNlcy5jb20=/index.html",
  "showcase/after/d3d3LmNvZGVwcmVuZXVyLmlv/index.html",
  "showcase/after/d3d3LndlYnN0YWNrZGV2LmNvbQ==/index.html",
  "showcase/after/dG91bG91c2Vqcy5mcmFuY2Vqcy5vcmc=/index.html",
  "showcase/after/dGhlZHVrZS5hdA==/index.html",
  "showcase/after/dmluc3BlZS5tZQ==/index.html",
  "showcase/after/emVuaWthLmNvbQ==/index.html",
  "showcase/after/Y21pY2hlbC5pbw==/index.html",
  "showcase/after/Y2VkLnBob3Rv/index.html",
  "showcase/after/Y3BwLmRheW5oYXVob2MuY29t/index.html",
  "showcase/after/Y3NzbmFuby5jbw==/index.html",
  "showcase/after/YW50aG9ueWR1Z29pcy5jb21fX3BvbHlub21pYw==/index.html",
  "showcase/after/YWNyb3dpdGhicmlhbi5jb20=/index.html",
  "showcase/after/Z2V0b3JiaXQuaW8=/index.html",
  "showcase/after/ZHJhd24taW4=/index.html",
  "showcase/after/ZWxkYXJsYWJzLmNvbQ==/index.html",
  "showcase/index.html",
  "showcase/submit/index.html",
  "showcase/tag/blog/after/a2hvYW5ndXllbi5tZQ==/index.html",
  "showcase/tag/blog/after/a2V2aW45NDA3MjYuZ2l0aHViLmlv/index.html",
  "showcase/tag/blog/after/aGF5d2lyZXouY29t/index.html",
  "showcase/tag/blog/after/d3d3LmFncmlzdGEuY29t/index.html",
  "showcase/tag/blog/after/d3d3LmJvc2NuZWdyZS12YWNhbmNlcy5jb20=/index.html",
  "showcase/tag/blog/after/d3d3LmNvZGVwcmVuZXVyLmlv/index.html",
  "showcase/tag/blog/after/d3d3LndlYnN0YWNrZGV2LmNvbQ==/index.html",
  "showcase/tag/blog/after/dGhlZHVrZS5hdA==/index.html",
  "showcase/tag/blog/after/dmluc3BlZS5tZQ==/index.html",
  "showcase/tag/blog/after/Y21pY2hlbC5pbw==/index.html",
  "showcase/tag/blog/after/Y2VkLnBob3Rv/index.html",
  "showcase/tag/blog/after/YWNyb3dpdGhicmlhbi5jb20=/index.html",
  "showcase/tag/blog/index.html",
  "showcase/tag/business/after/bmhha2hvYXBodW9jYmluaC5jb20=/index.html",
  "showcase/tag/business/after/d3d3LmFncmlzdGEuY29t/index.html",
  "showcase/tag/business/after/d3d3LmJvc2NuZWdyZS12YWNhbmNlcy5jb20=/index.html",
  "showcase/tag/business/after/emVuaWthLmNvbQ==/index.html",
  "showcase/tag/business/after/Y2VkLnBob3Rv/index.html",
  "showcase/tag/business/after/YWNyb3dpdGhicmlhbi5jb20=/index.html",
  "showcase/tag/business/after/Z2V0b3JiaXQuaW8=/index.html",
  "showcase/tag/business/after/ZWxkYXJsYWJzLmNvbQ==/index.html",
  "showcase/tag/business/index.html",
  "showcase/tag/community/after/c2VhYWRlMjAxNi52bg==/index.html",
  "showcase/tag/community/after/cHV0YWluZGVjb2RlLmlv/index.html",
  "showcase/tag/community/after/dG91bG91c2Vqcy5mcmFuY2Vqcy5vcmc=/index.html",
  "showcase/tag/community/after/ZHJhd24taW4=/index.html",
  "showcase/tag/community/index.html",
  "showcase/tag/docs/after/aXN0YW5idWwuanMub3Jn/index.html",
  "showcase/tag/docs/after/c2VydmVybGVzcy5jb20=/index.html",
  "showcase/tag/docs/after/c3R5bGVsaW50Lmlv/index.html",
  "showcase/tag/docs/after/cG9zdGNzcy5vcmc=/index.html",
  "showcase/tag/docs/after/cGhlbm9taWMuaW8=/index.html",
  "showcase/tag/docs/after/d3d3LmFncmlzdGEuY29t/index.html",
  "showcase/tag/docs/after/Y3NzbmFuby5jbw==/index.html",
  "showcase/tag/docs/after/YW50aG9ueWR1Z29pcy5jb21fX3BvbHlub21pYw==/index.html",
  "showcase/tag/docs/index.html",
  "showcase/tag/event/after/c2NhbGVjb25mY28uY29t/index.html",
  "showcase/tag/event/after/c2VhYWRlMjAxNi52bg==/index.html",
  "showcase/tag/event/after/dG91bG91c2Vqcy5mcmFuY2Vqcy5vcmc=/index.html",
  "showcase/tag/event/after/YWNyb3dpdGhicmlhbi5jb20=/index.html",
  "showcase/tag/event/index.html",
  "showcase/tag/learning/after/a2V2aW45NDA3MjYuZ2l0aHViLmlv/index.html",
  "showcase/tag/learning/after/aGF5d2lyZXouY29t/index.html",
  "showcase/tag/learning/after/c2NhbGVjb25mY28uY29t/index.html",
  "showcase/tag/learning/after/cHV0YWluZGVjb2RlLmlv/index.html",
  "showcase/tag/learning/after/cmVhY3RjbGFzcy5jb20=/index.html",
  "showcase/tag/learning/after/d3d3LndlYnN0YWNrZGV2LmNvbQ==/index.html",
  "showcase/tag/learning/after/Y21pY2hlbC5pbw==/index.html",
  "showcase/tag/learning/after/Y3BwLmRheW5oYXVob2MuY29t/index.html",
  "showcase/tag/learning/after/YWNyb3dpdGhicmlhbi5jb20=/index.html",
  "showcase/tag/learning/index.html",
  "showcase/tag/multi-languages/after/bW9vbm1haWwuaW9fX2FtYXpvbl9fc2VzX19lbWFpbF9fbWFya2V0aW5n/index.html",
  "showcase/tag/multi-languages/after/cHV0YWluZGVjb2RlLmlv/index.html",
  "showcase/tag/multi-languages/after/d3d3LmJvc2NuZWdyZS12YWNhbmNlcy5jb20=/index.html",
  "showcase/tag/multi-languages/after/emVuaWthLmNvbQ==/index.html",
  "showcase/tag/multi-languages/index.html",
  "showcase/tag/open-source/after/a2V2aW45NDA3MjYuZ2l0aHViLmlv/index.html",
  "showcase/tag/open-source/after/aXN0YW5idWwuanMub3Jn/index.html",
  "showcase/tag/open-source/after/bm9yZHNvZnR3YXJlLmdpdGh1Yi5pbw==/index.html",
  "showcase/tag/open-source/after/bW9vbm1haWwuaW9fX2FtYXpvbl9fc2VzX19lbWFpbF9fbWFya2V0aW5n/index.html",
  "showcase/tag/open-source/after/c2NhbGVjb25mY28uY29t/index.html",
  "showcase/tag/open-source/after/c2VydmVybGVzcy5jb20=/index.html",
  "showcase/tag/open-source/after/c3R5bGVsaW50Lmlv/index.html",
  "showcase/tag/open-source/after/cG9zdGNzcy5vcmc=/index.html",
  "showcase/tag/open-source/after/cGhlbm9taWMuaW8=/index.html",
  "showcase/tag/open-source/after/cHV0YWluZGVjb2RlLmlv/index.html",
  "showcase/tag/open-source/after/Y3NzbmFuby5jbw==/index.html",
  "showcase/tag/open-source/after/YW50aG9ueWR1Z29pcy5jb21fX3BvbHlub21pYw==/index.html",
  "showcase/tag/open-source/index.html",
  "showcase/tag/service/after/bW9vbm1haWwuaW9fX2FtYXpvbl9fc2VzX19lbWFpbF9fbWFya2V0aW5n/index.html",
  "showcase/tag/service/after/c2VydmVybGVzcy5jb20=/index.html",
  "showcase/tag/service/after/d3d3LmJsdWVqYXl3aXJlbGVzcy5jb20=/index.html",
  "showcase/tag/service/after/Z2V0b3JiaXQuaW8=/index.html",
  "showcase/tag/service/index.html",
]
`;

exports[`should build docs correctly 2`] = `
Array [
  "phenomic/news/by-default/1/desc/limit-10.json",
  "phenomic/news/by-default/1/desc/limit-10/after-MjAxNy8wNi9pbnRyb2R1Y2luZy0xLjAuMC1hbHBoYQ==.json",
  "phenomic/news/item/2017/06/introducing-1.0.0-alpha.json",
  "phenomic/showcase-entries/by-default/1/asc/limit-10.json",
  "phenomic/showcase-entries/by-default/1/asc/limit-10/after-a2hvYW5ndXllbi5tZQ==.json",
  "phenomic/showcase-entries/by-default/1/asc/limit-10/after-a2V2aW45NDA3MjYuZ2l0aHViLmlv.json",
  "phenomic/showcase-entries/by-default/1/asc/limit-10/after-aGF5d2lyZXouY29t.json",
  "phenomic/showcase-entries/by-default/1/asc/limit-10/after-aXN0YW5idWwuanMub3Jn.json",
  "phenomic/showcase-entries/by-default/1/asc/limit-10/after-bm9yZHNvZnR3YXJlLmdpdGh1Yi5pbw==.json",
  "phenomic/showcase-entries/by-default/1/asc/limit-10/after-bmhha2hvYXBodW9jYmluaC5jb20=.json",
  "phenomic/showcase-entries/by-default/1/asc/limit-10/after-bW9vbm1haWwuaW9fX2FtYXpvbl9fc2VzX19lbWFpbF9fbWFya2V0aW5n.json",
  "phenomic/showcase-entries/by-default/1/asc/limit-10/after-c2NhbGVjb25mY28uY29t.json",
  "phenomic/showcase-entries/by-default/1/asc/limit-10/after-c2VhYWRlMjAxNi52bg==.json",
  "phenomic/showcase-entries/by-default/1/asc/limit-10/after-c2VydmVybGVzcy5jb20=.json",
  "phenomic/showcase-entries/by-default/1/asc/limit-10/after-c3R5bGVsaW50Lmlv.json",
  "phenomic/showcase-entries/by-default/1/asc/limit-10/after-cG9zdGNzcy5vcmc=.json",
  "phenomic/showcase-entries/by-default/1/asc/limit-10/after-cGhlbm9taWMuaW8=.json",
  "phenomic/showcase-entries/by-default/1/asc/limit-10/after-cHV0YWluZGVjb2RlLmlv.json",
  "phenomic/showcase-entries/by-default/1/asc/limit-10/after-cmVhY3RjbGFzcy5jb20=.json",
  "phenomic/showcase-entries/by-default/1/asc/limit-10/after-d3d3LmFncmlzdGEuY29t.json",
  "phenomic/showcase-entries/by-default/1/asc/limit-10/after-d3d3LmJsdWVqYXl3aXJlbGVzcy5jb20=.json",
  "phenomic/showcase-entries/by-default/1/asc/limit-10/after-d3d3LmJvc2NuZWdyZS12YWNhbmNlcy5jb20=.json",
  "phenomic/showcase-entries/by-default/1/asc/limit-10/after-d3d3LmNvZGVwcmVuZXVyLmlv.json",
  "phenomic/showcase-entries/by-default/1/asc/limit-10/after-d3d3LndlYnN0YWNrZGV2LmNvbQ==.json",
  "phenomic/showcase-entries/by-default/1/asc/limit-10/after-dG91bG91c2Vqcy5mcmFuY2Vqcy5vcmc=.json",
  "phenomic/showcase-entries/by-default/1/asc/limit-10/after-dGhlZHVrZS5hdA==.json",
  "phenomic/showcase-entries/by-default/1/asc/limit-10/after-dmluc3BlZS5tZQ==.json",
  "phenomic/showcase-entries/by-default/1/asc/limit-10/after-emVuaWthLmNvbQ==.json",
  "phenomic/showcase-entries/by-default/1/asc/limit-10/after-Y21pY2hlbC5pbw==.json",
  "phenomic/showcase-entries/by-default/1/asc/limit-10/after-Y2VkLnBob3Rv.json",
  "phenomic/showcase-entries/by-default/1/asc/limit-10/after-Y3BwLmRheW5oYXVob2MuY29t.json",
  "phenomic/showcase-entries/by-default/1/asc/limit-10/after-Y3NzbmFuby5jbw==.json",
  "phenomic/showcase-entries/by-default/1/asc/limit-10/after-YW50aG9ueWR1Z29pcy5jb21fX3BvbHlub21pYw==.json",
  "phenomic/showcase-entries/by-default/1/asc/limit-10/after-YWNyb3dpdGhicmlhbi5jb20=.json",
  "phenomic/showcase-entries/by-default/1/asc/limit-10/after-Z2V0b3JiaXQuaW8=.json",
  "phenomic/showcase-entries/by-default/1/asc/limit-10/after-ZHJhd24taW4=.json",
  "phenomic/showcase-entries/by-default/1/asc/limit-10/after-ZWxkYXJsYWJzLmNvbQ==.json",
  "phenomic/showcase-entries/by-showcaseTags/blog/asc/limit-10.json",
  "phenomic/showcase-entries/by-showcaseTags/blog/asc/limit-10/after-a2hvYW5ndXllbi5tZQ==.json",
  "phenomic/showcase-entries/by-showcaseTags/blog/asc/limit-10/after-a2V2aW45NDA3MjYuZ2l0aHViLmlv.json",
  "phenomic/showcase-entries/by-showcaseTags/blog/asc/limit-10/after-aGF5d2lyZXouY29t.json",
  "phenomic/showcase-entries/by-showcaseTags/blog/asc/limit-10/after-d3d3LmFncmlzdGEuY29t.json",
  "phenomic/showcase-entries/by-showcaseTags/blog/asc/limit-10/after-d3d3LmJvc2NuZWdyZS12YWNhbmNlcy5jb20=.json",
  "phenomic/showcase-entries/by-showcaseTags/blog/asc/limit-10/after-d3d3LmNvZGVwcmVuZXVyLmlv.json",
  "phenomic/showcase-entries/by-showcaseTags/blog/asc/limit-10/after-d3d3LndlYnN0YWNrZGV2LmNvbQ==.json",
  "phenomic/showcase-entries/by-showcaseTags/blog/asc/limit-10/after-dGhlZHVrZS5hdA==.json",
  "phenomic/showcase-entries/by-showcaseTags/blog/asc/limit-10/after-dmluc3BlZS5tZQ==.json",
  "phenomic/showcase-entries/by-showcaseTags/blog/asc/limit-10/after-Y21pY2hlbC5pbw==.json",
  "phenomic/showcase-entries/by-showcaseTags/blog/asc/limit-10/after-Y2VkLnBob3Rv.json",
  "phenomic/showcase-entries/by-showcaseTags/blog/asc/limit-10/after-YWNyb3dpdGhicmlhbi5jb20=.json",
  "phenomic/showcase-entries/by-showcaseTags/business/asc/limit-10.json",
  "phenomic/showcase-entries/by-showcaseTags/business/asc/limit-10/after-bmhha2hvYXBodW9jYmluaC5jb20=.json",
  "phenomic/showcase-entries/by-showcaseTags/business/asc/limit-10/after-d3d3LmFncmlzdGEuY29t.json",
  "phenomic/showcase-entries/by-showcaseTags/business/asc/limit-10/after-d3d3LmJvc2NuZWdyZS12YWNhbmNlcy5jb20=.json",
  "phenomic/showcase-entries/by-showcaseTags/business/asc/limit-10/after-emVuaWthLmNvbQ==.json",
  "phenomic/showcase-entries/by-showcaseTags/business/asc/limit-10/after-Y2VkLnBob3Rv.json",
  "phenomic/showcase-entries/by-showcaseTags/business/asc/limit-10/after-YWNyb3dpdGhicmlhbi5jb20=.json",
  "phenomic/showcase-entries/by-showcaseTags/business/asc/limit-10/after-Z2V0b3JiaXQuaW8=.json",
  "phenomic/showcase-entries/by-showcaseTags/business/asc/limit-10/after-ZWxkYXJsYWJzLmNvbQ==.json",
  "phenomic/showcase-entries/by-showcaseTags/community/asc/limit-10.json",
  "phenomic/showcase-entries/by-showcaseTags/community/asc/limit-10/after-c2VhYWRlMjAxNi52bg==.json",
  "phenomic/showcase-entries/by-showcaseTags/community/asc/limit-10/after-cHV0YWluZGVjb2RlLmlv.json",
  "phenomic/showcase-entries/by-showcaseTags/community/asc/limit-10/after-dG91bG91c2Vqcy5mcmFuY2Vqcy5vcmc=.json",
  "phenomic/showcase-entries/by-showcaseTags/community/asc/limit-10/after-ZHJhd24taW4=.json",
  "phenomic/showcase-entries/by-showcaseTags/docs/asc/limit-10.json",
  "phenomic/showcase-entries/by-showcaseTags/docs/asc/limit-10/after-aXN0YW5idWwuanMub3Jn.json",
  "phenomic/showcase-entries/by-showcaseTags/docs/asc/limit-10/after-c2VydmVybGVzcy5jb20=.json",
  "phenomic/showcase-entries/by-showcaseTags/docs/asc/limit-10/after-c3R5bGVsaW50Lmlv.json",
  "phenomic/showcase-entries/by-showcaseTags/docs/asc/limit-10/after-cG9zdGNzcy5vcmc=.json",
  "phenomic/showcase-entries/by-showcaseTags/docs/asc/limit-10/after-cGhlbm9taWMuaW8=.json",
  "phenomic/showcase-entries/by-showcaseTags/docs/asc/limit-10/after-d3d3LmFncmlzdGEuY29t.json",
  "phenomic/showcase-entries/by-showcaseTags/docs/asc/limit-10/after-Y3NzbmFuby5jbw==.json",
  "phenomic/showcase-entries/by-showcaseTags/docs/asc/limit-10/after-YW50aG9ueWR1Z29pcy5jb21fX3BvbHlub21pYw==.json",
  "phenomic/showcase-entries/by-showcaseTags/event/asc/limit-10.json",
  "phenomic/showcase-entries/by-showcaseTags/event/asc/limit-10/after-c2NhbGVjb25mY28uY29t.json",
  "phenomic/showcase-entries/by-showcaseTags/event/asc/limit-10/after-c2VhYWRlMjAxNi52bg==.json",
  "phenomic/showcase-entries/by-showcaseTags/event/asc/limit-10/after-dG91bG91c2Vqcy5mcmFuY2Vqcy5vcmc=.json",
  "phenomic/showcase-entries/by-showcaseTags/event/asc/limit-10/after-YWNyb3dpdGhicmlhbi5jb20=.json",
  "phenomic/showcase-entries/by-showcaseTags/learning/asc/limit-10.json",
  "phenomic/showcase-entries/by-showcaseTags/learning/asc/limit-10/after-a2V2aW45NDA3MjYuZ2l0aHViLmlv.json",
  "phenomic/showcase-entries/by-showcaseTags/learning/asc/limit-10/after-aGF5d2lyZXouY29t.json",
  "phenomic/showcase-entries/by-showcaseTags/learning/asc/limit-10/after-c2NhbGVjb25mY28uY29t.json",
  "phenomic/showcase-entries/by-showcaseTags/learning/asc/limit-10/after-cHV0YWluZGVjb2RlLmlv.json",
  "phenomic/showcase-entries/by-showcaseTags/learning/asc/limit-10/after-cmVhY3RjbGFzcy5jb20=.json",
  "phenomic/showcase-entries/by-showcaseTags/learning/asc/limit-10/after-d3d3LndlYnN0YWNrZGV2LmNvbQ==.json",
  "phenomic/showcase-entries/by-showcaseTags/learning/asc/limit-10/after-Y21pY2hlbC5pbw==.json",
  "phenomic/showcase-entries/by-showcaseTags/learning/asc/limit-10/after-Y3BwLmRheW5oYXVob2MuY29t.json",
  "phenomic/showcase-entries/by-showcaseTags/learning/asc/limit-10/after-YWNyb3dpdGhicmlhbi5jb20=.json",
  "phenomic/showcase-entries/by-showcaseTags/multi-languages/asc/limit-10.json",
  "phenomic/showcase-entries/by-showcaseTags/multi-languages/asc/limit-10/after-bW9vbm1haWwuaW9fX2FtYXpvbl9fc2VzX19lbWFpbF9fbWFya2V0aW5n.json",
  "phenomic/showcase-entries/by-showcaseTags/multi-languages/asc/limit-10/after-cHV0YWluZGVjb2RlLmlv.json",
  "phenomic/showcase-entries/by-showcaseTags/multi-languages/asc/limit-10/after-d3d3LmJvc2NuZWdyZS12YWNhbmNlcy5jb20=.json",
  "phenomic/showcase-entries/by-showcaseTags/multi-languages/asc/limit-10/after-emVuaWthLmNvbQ==.json",
  "phenomic/showcase-entries/by-showcaseTags/open-source/asc/limit-10.json",
  "phenomic/showcase-entries/by-showcaseTags/open-source/asc/limit-10/after-a2V2aW45NDA3MjYuZ2l0aHViLmlv.json",
  "phenomic/showcase-entries/by-showcaseTags/open-source/asc/limit-10/after-aXN0YW5idWwuanMub3Jn.json",
  "phenomic/showcase-entries/by-showcaseTags/open-source/asc/limit-10/after-bm9yZHNvZnR3YXJlLmdpdGh1Yi5pbw==.json",
  "phenomic/showcase-entries/by-showcaseTags/open-source/asc/limit-10/after-bW9vbm1haWwuaW9fX2FtYXpvbl9fc2VzX19lbWFpbF9fbWFya2V0aW5n.json",
  "phenomic/showcase-entries/by-showcaseTags/open-source/asc/limit-10/after-c2NhbGVjb25mY28uY29t.json",
  "phenomic/showcase-entries/by-showcaseTags/open-source/asc/limit-10/after-c2VydmVybGVzcy5jb20=.json",
  "phenomic/showcase-entries/by-showcaseTags/open-source/asc/limit-10/after-c3R5bGVsaW50Lmlv.json",
  "phenomic/showcase-entries/by-showcaseTags/open-source/asc/limit-10/after-cG9zdGNzcy5vcmc=.json",
  "phenomic/showcase-entries/by-showcaseTags/open-source/asc/limit-10/after-cGhlbm9taWMuaW8=.json",
  "phenomic/showcase-entries/by-showcaseTags/open-source/asc/limit-10/after-cHV0YWluZGVjb2RlLmlv.json",
  "phenomic/showcase-entries/by-showcaseTags/open-source/asc/limit-10/after-Y3NzbmFuby5jbw==.json",
  "phenomic/showcase-entries/by-showcaseTags/open-source/asc/limit-10/after-YW50aG9ueWR1Z29pcy5jb21fX3BvbHlub21pYw==.json",
  "phenomic/showcase-entries/by-showcaseTags/service/asc/limit-10.json",
  "phenomic/showcase-entries/by-showcaseTags/service/asc/limit-10/after-bW9vbm1haWwuaW9fX2FtYXpvbl9fc2VzX19lbWFpbF9fbWFya2V0aW5n.json",
  "phenomic/showcase-entries/by-showcaseTags/service/asc/limit-10/after-c2VydmVybGVzcy5jb20=.json",
  "phenomic/showcase-entries/by-showcaseTags/service/asc/limit-10/after-d3d3LmJsdWVqYXl3aXJlbGVzcy5jb20=.json",
  "phenomic/showcase-entries/by-showcaseTags/service/asc/limit-10/after-Z2V0b3JiaXQuaW8=.json",
  "phenomic/showcase/item/submit.json",
]
`;
